.github/workflows/ci.yml

```diff
@@ -285,9 +308,12 @@ jobs:
 
       - run: make flutter.pub
 
+      # TODO: Use `split-debug-info` when Sentry supports Linux debug symbols.
+      #       https://github.com/getsentry/sentry-dart/issues/433
       - run: make flutter.build platform=linux
                   dart-env='SOCAPP_HTTP_URL=${{ secrets.BACKEND_URL }}
                             SOCAPP_HTTP_PORT=${{ secrets.BACKEND_PORT }}
+                            SOCAPP_SENTRY_DSN=${{ secrets.SENTRY_DSN }}
```

Comment:
@SleepySquash don't we need debug symbols for Linux the same way we do for Android/iOS/macOS?